WordPress-VIP-Go/ruleset.xml

```diff
@@ -80,56 +80,56 @@
 	<rule ref="WordPress.WP.PostsPerPage.posts_per_page_posts_per_page">
 		<message>Having more than 100 posts returned per page can lead to severe performance problems.</message>
 	</rule>
-	<rule ref="WordPressVIPMinimum.Filters.RestrictedHook.UploadMimes">
+	<rule ref="WordPressVIPMinimum.Hooks.RestrictedHook.UploadMimes">
```

Comment:
We said that `RestrictedHook` was going to be pluralised into `RestrictedHooks`.